File: Project-Euler/Solutions-Simon/euler5/euler5.py
def divisible():
	evenlyDivis = True
	n = 0
	for x in xrange(21, 500000000):
		evenlyDivis = True
		if x == 500000000:
			print "Done"
		for i in range(2, 20):
			n = x % i
			if not(n == 0):
				evenlyDivis = False
				break
		if evenlyDivis:
			print "%d is evenly divisible" % x

divisible()

File: Project-Euler/Solutions-Simon/euler5/euler5.java
public class euler5 
{
	public static void main(String[] args)
	{
		divisible();
	}
	
	private static void divisible()
	{
		boolean evenlyDivis = true;
		long n = 0;
		for(long x = 21; x <= 500000000l; x++)
		{
			evenlyDivis = true;
			if(x == 500000000l)
			{
				System.out.println("Done");
			}
			for(long i = 2; i <= 20; i++)
			{
				n = x % i;
				if(n != 0)
				{
					evenlyDivis = false;
					break;
				}
			}
			if(evenlyDivis)
			{
				System.out.println(x + " is evenly divisible");
			}
		}
	}
}
